scene/main/window.cpp

```diff
@@ -1748,7 +1748,9 @@ Rect2i Window::fit_rect_in_parent(Rect2i p_rect, const Rect2i &p_parent_rect) co
 
 Size2 Window::get_contents_minimum_size() const {
 	ERR_READ_THREAD_GUARD_V(Size2());
-	return _get_contents_minimum_size();
+	Vector2 ms = _get_contents_minimum_size();
+	GDVIRTUAL_CALL(_get_contents_minimum_size, ms);
```

Comment:
Isn't it going to be wasteful to always run the default `_get_contents_minimum_size()` before the virtual call? How heavy is that call?

Not tested, but this might be better for performance:
```suggestion
	Vector2 ms;
	if (GDVIRTUAL_CALL(_get_contents_minimum_size, ms) {
		return ms;
	}
	return _get_contents_minimum_size();
```